samples/stored-procedures/clear.js

```diff
@@ -0,0 +1,121 @@
+/**

```

Comment:
Can this be renamed to bulkDelete_underscoreAPI.js to make it clear this is similar to bulkDelete.js but using underscore API?